tools/sdk-cli/Cargo.toml

```diff
@@ -16,4 +14,8 @@ serde_json = "1.0"
 tiny-bip39 = "1"
 tokio = { version = "1", features = ["rt-multi-thread"] }
 serde = { version = "1.0", features = ["derive"] }
-clap = { version = "4", features = ["derive"] }
\ No newline at end of file
+clap = { version = "4", features = ["derive"] }
+
+[profile.dev]
+debug = 0
+strip = "debuginfo"
```

Comment:
Intuitively it seems to me that debuginfo belongs to dev profile (othewise where elsewhere?).
A simple solution is to come up with a custom profile that a developer would have to include in the cargo build command?
Saying all that, I have no strong opinion here so only listed my thoughts.

Reply:
@andrei-21 what are your thoughts on this?